docs/content/Header.mdx

```diff
@@ -54,25 +54,25 @@ All items directly under the Header component should be a `Header.Item` componen
 </Header>
 ```
 
-## Component props
+## Props
 
-### Header
+### `Header`
 
-| Name | Type              | Default | Description                          |
-| :--- | :---------------- | :-----: | :----------------------------------- |
-| sx   | SystemStyleObject |   {}    | Style to be applied to the component |
+<PropsTable>
+  <PropsTableSxRow />
+</PropsTable>
 
-### Header.Item
+### `Header.Item`
 
-| Name | Type              | Default | Description                                |
-| :--- | :---------------- | :-----: | :----------------------------------------- |
-| full | Boolean           |         | stretches item to fill the available space |
-| sx   | SystemStyleObject |   {}    | Style to be applied to the component       |
+<PropsTable>
+  <PropsTableRow name="full" type="string" description="Stretches item to fill the available space" />
+  <PropsTableSxRow />
+</PropsTable>
 
-### Header.Link
+### `Header.Link`
 
-| Name | Type              | Default | Description                          |
-| :--- | :---------------- | :-----: | :----------------------------------- |
-| as   | String            |         | sets the HTML tag for the component  |
-| href | String            |         | URL to be used for the Link          |
-| sx   | SystemStyleObject |   {}    | Style to be applied to the component |
+<PropsTable>
+  <PropsTableRow name="href" type="string" description="URL to be used for the Link" />
+  <PropsTableAsRow defaultElementType="a" />
+  <PropsTableSxRow />
```

Comment:
Does this component also pass props down to the `<a>` element?

Reply:
Looks like it does, updated with base props. Is that more suitable? 🤔

Reply:
Update: I've put a guard on `refType` so that row doesn't appear.